### PR TITLE
Added KubeCon news article w/signoff

### LINKED ISSUE #836: [NG] Add KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive news article
**Desired Behavior**
Add `KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive` news article present on the [current site](https://layer5.io/company/news/cncf-sig-network-intro-and-deep-dive) to layer5ng. Check[ #815](https://github.com/layer5io/layer5/issues/815) for more more details. 

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by anshdhinhgra47**
Signed-off-by: Ansh Dhingra <56676059+anshdhinhgra47@users.noreply.github.com>

**Description**
This PR has added the KubeCon news article to the new site
This PR fixes #836

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/news/2019-11-23-KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive/index.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+--
+title: "KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive"
+subtitle: ""
+date: 2019-11-23 12:15:05 +0000
+author: the Newsroom
+thumbnail: ./KubeCon_NA_2019_bg-01.svg
+eurl: https://landscape.cncf.io/category=cncf-members&format=logo-mode&grouping=category&selected=layer5-member&embed=yes
```
Comment (Tanuj22) [inline src/collections/news/2019-11-23-KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive/index.mdx:7]: This should be blank right?

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Nice work! The article is not visible in the news gird https://deploy-preview-874--layer5ng.netlify.app/news. 
probably because of the folder name. Change the folder name to `2019-11-23-cncf-sig-network-intro-and-deep-dive`

File: src/collections/news/2019-11-23-KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive/index.mdx (hunk 1 above)
Comment (anshdhinhgra47) [reply src/collections/news/2019-11-23-KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive/index.mdx:7]: I couldn't understand. I had followed the News Style Guide.

Comment (Tanuj22) [reply src/collections/news/2019-11-23-KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive/index.mdx:7]: Yes! But for this particular article, the eurl is null. So you can leave it as `eurl: ""`

Comment (anshdhinhgra47) [reply src/collections/news/2019-11-23-KubeCon NA: CNCF SIG Network Set Intro and Deep-Dive/index.mdx:7]: oh okay

**Comment by Tanuj22** [comment]
It's still not showing up. I wonder why :thinking:

**Comment by Tanuj22** [comment]
@anshdhinhgra47 Is it working for you in local?

**Comment by anshdhinhgra47** [comment]
yeah, it is working fine

**Comment by Tanuj22** [comment]
@anshdhinhgra47 there was a problem in the frontmatter.

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Good Work but some changes are needed - 
- The video is overflowing in mobile view 
Screenshot from 2020-10-02 19-11-57

- The "Releated news by ...." should be moved to the next line 
Screenshot from 2020-10-02 19-13-52

**Comment by anshdhinhgra47** [comment]
Sure! I'll just fix it. Can do it somewhere in the next week? I have my mid-sem exams coming up.

**Comment by leecalcote** [comment]
@anshdhinhgra47 thanks for setting expectations on timing. Fingers-crossed for your exams.

**Comment by anshdhinhgra47** [comment]
Thank you so much, Lee! I'll be working on this issue now. My exams did go pretty well.

**Comment by Tanuj22** [comment]
@anshdhinhgra47 are you working on this?

**Comment by anshdhinhgra47** [comment]
Yes

**Comment by anshdhinhgra47** [comment]
I am not able to force push the commits. Could you help me with that ?

**Comment by Tanuj22** [comment]
What's it showing?

**Comment by anshdhinhgra47** [comment]
This is the error:


fatal: The current branch anshdhinhgra47/add-KubCon-news-article has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin anshdhinhgra47/add-KubCon-news-article

**Comment by Jashpatel1** [comment]
> This is the error:
> 
> fatal: The current branch anshdhinhgra47/add-KubCon-news-article has no upstream branch.
> To push the current branch and set the remote as upstream, use
> 
> ```
> git push --set-upstream origin anshdhinhgra47/add-KubCon-news-article
> ```

Are you doing git push origin anshdhinhgra47/add-KubCon-news-article?

**Comment by anshdhinhgra47** [comment]
I am trying to do `git push --force`

**Comment by Tanuj22** [comment]
You have to do `git push origin anshdhinhgra47/add-KubCon-news-article`

**Comment by anshdhinhgra47** [comment]
This is what's coming up: 


To https://github.com/anshdhinhgra47/layer5.git
 ! [rejected]          anshdhinhgra47/add-KubCon-news-article -> anshdhinhgra47/add-KubCon-news-article (non-fast-forward)
error: failed to push some refs to 'https://github.com/anshdhinhgra47/layer5.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

**Comment by leecalcote** [comment]
You’ll want to sync your fork as it is out of date from changes that have transpired in the time that you have formed the repo.

**Comment by anshdhinhgra47** [comment]
Thanks @leecalcote !!

**Comment by Tanuj22** [comment]
@anshdhinhgra47 is this PR ready for review?

File: src/collections/news/2019-11-23-cncf-sig-network-intro-and-deep-dive/index.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+--
```
Comment (Tanuj22) [inline src/collections/news/2019-11-23-cncf-sig-network-intro-and-deep-dive/index.mdx:1]:
```suggestion
---
```

**Comment by anshdhinhgra47** [comment]
Yes

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Still overflowing - 
Screenshot from 2020-10-18 02-40-54

**Comment by Tanuj22** [comment]
Also please fix the DCO - https://github.com/layer5io/layer5/pull/874/checks?check_run_id=1269210700

**Comment by anshdhinhgra47** [comment]
I had fixed the screen resolution for small screens

**Comment by Tanuj22** [comment]
> I had fixed the screen resolution for small screens

Well it's not fixed

**Review by Tanuj22 — APPROVED** [review]
I am fixed it. Thanks @anshdhinhgra47

**Comment by anshdhinhgra47** [comment]
I am really sorry I just saw the message. Was stuck up in a meeting. I really apologize for letting you down and for not being able to perform to your standards. I am really sorry for not being able to solve this issue.

**Pull request merged**